docs/guides/postgres/backup/kubestash/logical/index.md

```diff
@@ -0,0 +1,760 @@
+```
+
+> Note: KubeStash creates a `Snapshot` with the following labels:
+> - `kubedb.com/db-version: <db-version>`
+> - `kubestash.com/app-ref-kind: <target-kind>`
+> - `kubestash.com/app-ref-name: <target-name>`
+> - `kubestash.com/app-ref-namespace: <target-namespace>`
+> - `kubestash.com/repo-name: <repository-name>`
+>
+> These labels can be used to watch only the `Snapshot`s related to our target Database or `Repository`.
+
+If we check the YAML of the `Snapshot`, we can find the information about the backed up components of the Database.
+
+```bash
+$ kubectl get snapshots -n demo gcs-postgres-repo-sample-postgres-backup-frequent-backup-1725449400 -oyaml
+```
+
+```yaml
+apiVersion: storage.kubestash.com/v1alpha1
+kind: Snapshot
+metadata:
+  creationTimestamp: "2024-09-04T11:30:00Z"
+  finalizers:
+  - kubestash.com/cleanup
+  generation: 1
+  labels:
+    kubedb.com/db-version: "16.1"
+    kubestash.com/app-ref-kind: Postgres
+    kubestash.com/app-ref-name: sample-postgres
+    kubestash.com/app-ref-namespace: demo
+    kubestash.com/repo-name: gcs-postgres-repo
+  name: gcs-postgres-repo-sample-postgreckup-frequent-backup-1725449400
+  namespace: demo
+  ownerReferences:
+  - apiVersion: storage.kubestash.com/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: Repository
+    name: gcs-postgres-repo
+    uid: 1009bd4a-b211-49f1-a64c-3c979c699a81
+  resourceVersion: "253523"
+  uid: c6757c49-e13b-4a36-9f7d-64eae350423f
+spec:
+  appRef:
+    apiGroup: kubedb.com
+    kind: Postgres
+    name: sample-postgres
+    namespace: demo
+  backupSession: sample-postgres-backup-frequent-backup-1725449400
+  deletionPolicy: Delete
+  repository: gcs-postgres-repo
+  session: frequent-backup
+  snapshotID: 01J6YCRWEWAKACMGZYR2R7YJ5C
+  type: FullBackup
+  version: v1
+status:
+  components:
+    dump:
+      driver: Restic
+      duration: 11.526138009s
+      integrity: true
+      path: repository/v1/frequent-backup/dump
+      phase: Succeeded
+      resticStats:
+      - hostPath: dumpfile.sql
+        id: 008eb87193e7db112e9ad8f42c9302c851a1fbacb7165a5cb3aa2d27dd210764
+        size: 3.345 KiB
+        uploaded: 299 B
+      size: 2.202 KiB
+  conditions:
+  - lastTransitionTime: "2024-09-04T11:30:00Z"
+    message: Recent snapshot list updated successfully
+    reason: SuccessfullyUpdatedRecentSnapshotList
+    status: "True"
+    type: RecentSnapshotListUpdated
+  - lastTransitionTime: "2024-09-04T11:30:32Z"
+    message: Metadata uploaded to backend successfully
+    reason: SuccessfullyUploadedSnapshotMetadata
+    status: "True"
+    type: SnapshotMetadataUploaded
+  integrity: true
+  phase: Succeeded
+  size: 2.201 KiB
+  snapshotTime: "2024-09-04T11:30:00Z"
+  totalComponents: 1
+```
+
+> KubeStash uses the `pg_dump` command to take backups of target PostgreSQL databases. Therefore, the component name for `logical backups` is set as `dump`.
```

Comment:
Change this. Maybe we can write:

KubeStash uses a logical backup approach to take backups of target PostgreSQL databases. Therefore, the component name for logical backups is set as `dump`.